daliuge-engine/dlg/manager/proc_daemon.py

```diff
@@ -84,6 +84,7 @@ def __init__(self, master=False, noNM=False, disable_zeroconf=False, verbosity=0
         self._zeroconf = None if disable_zeroconf else zc.Zeroconf()
         self._nm_info = None
         self._mm_browser = None
+        self._mm_dim_browser = None
```

Comment:
Since this is `mm_dim_browser` maybe the one above should also be renamed to `mm_nm_browser`